README.md

```diff
@@ -2,29 +2,21 @@ Confluent's Python Client for Apache Kafka<sup>TM</sup>
 =======================================================
 
 **confluent-kafka-python** is Confluent's Python client for [Apache Kafka](http://kafka.apache.org/) and the
-[Confluent Platform](https://www.confluent.io/product/compare/).
+[Confluent Platform](https://www.confluent.io/product/compare/). It provides a high-level Producer, Consumer
+and AdminClient analogous to the Java client which are backwards and forwards compatible with all
```

Comment:
I'm not sure I like the `analogous to the Java client` bit, do Python users care?
And analogous is a foreign word for most non-native english speakers

Reply:
i'd agree with that.